src/main/java/com/aerospike/mapper/tools/configuration/ReferenceConfig.java

```diff
@@ -7,12 +7,15 @@ public class ReferenceConfig {
     private Boolean lazy;
     private Boolean batchLoad;
 
-    public ReferenceConfig() {}
-    public ReferenceConfig(ReferenceType type, boolean lazy) {
+    public ReferenceConfig() {
+    }
+
+    public ReferenceConfig(ReferenceType type, boolean lazy, boolean batchLoad) {
```

Comment:
@tim-aero can a user call this constructor or is it completely internal? if so - it is a breaking change (they need to pass batchLoad as well)

Reply:
Update:
The answer is no, it is used internally